### PR TITLE
Introduce DSCC

### PR DESCRIPTION
**Pull request by mschoenlaub**
See https://developers.google.com/datastudio/connector/local-development for rationale. Testing is becoming as easy as `yarn push && yarn try_latest` .

Deploy with `yarn update_production`
Open script with `yarn open`

had to copy the sourcefiles though (and apparently prettifying is automatically done).

### CODE REVIEW

**Comment by Tigraine** [comment]
This is cool. Need to look into it tomorrow

**Review by rolandgriesser — APPROVED** [review]
Really cool tool, makes it way easier to deploy! 👍

File: src/main.js [hunk 1]
```diff
@@ -0,0 +1,543 @@
...
+      throw new Error("Limit has to be a numeric value.");
+    }
+
+    if (configParams.offset && isNaN(parseInt(configParams.offset))) {
+      throw new Error("Offset has to be a numeric value.");
+    }
+
+    if (configParams.aggregation === "percentile") {
+      var percentile = parseInt(configParams.percentile);
+      if (isNaN(percentile) || percentile < 0 || percentile > 100) {
```
Comment (peterpf) [inline src/main.js:406]:
```suggestion
      if (isNaN(percentile) || percentile < 0 || percentile >= 100) {
```

File: src/main.js [hunk 2]
```diff
@@ -0,0 +1,543 @@
...
+      "VIDEO_WINDOW_HEIGHT",
+      "VIDEO_WINDOW_WIDTH",
+      "VIDEOTIME_END",
+      "VIDEOTIME_START",
+      "OPERATINGSYSTEM",
+      "OPERATINGSYSTEM_VERSION_MAJOR",
+      "PAGE_LOAD_TIME",
+      "PAGE_LOAD_TYPE",
+      "PROG_URL"
+    ];
```
Comment (peterpf) [inline src/main.js:96]: Guess we should update the list (in another PR)

**Review by peterpf — APPROVED** [review]
Added 1-2 comments but ready to go :+1:

**Pull request merged**